src/qibolab/qubits.py

```diff
@@ -73,10 +78,22 @@ def __post_init__(self):
 
     @property
     def channels(self):
-        for channel in [self.readout, self.feedback, self.drive, self.flux, self.twpa]:
+        for name in CHANNEL_NAMES:
+            channel = getattr(self, name)
             if channel is not None:
                 yield channel
 
+    @property
+    def characterization(self):
+        """Dictionary containing characterization parameters."""
+        exclude_fields = CHANNEL_NAMES + ("name", "flux_coupler", "native_gates")
+        data = {fld.name: getattr(self, fld.name) for fld in fields(self) if fld.name not in exclude_fields}
+        return data
```

Comment:
Stupid simplification:
```suggestion
        return {fld.name: getattr(self, fld.name) for fld in fields(self) if fld.name not in exclude_fields}
```